classes/user/form/RolesForm.php

```diff
@@ -43,7 +43,11 @@ public function fetch($request, $template = null, $display = false)
     {
         $templateMgr = TemplateManager::getManager($request);
 
-        $userGroupIds = UserGroup::getIdsByUserId($request->getUser()->getId());
+        $userGroupIds = UserGroup::query()
+            ->withUserIds([$request->getUser()->getId()])
+            ->get()
+            ->pluck('id')
+            ->toArray();
```

Comment:
@Vitaliy-1 without change, the profile --> roles tab is broken . and I dont see any static method `getIdsByUserId` defined for class `UserGroup` . Is this ok ?